Remove check for getopt_long and not existing source
files common/getopt.c and common/getopt1.c

--- a/configure.ac	2015-10-25 20:55:07.635988008 +0100
+++ b/configure.ac	2015-10-25 20:56:18.683993130 +0100
@@ -20,7 +20,6 @@
 
 AC_CHECK_HEADERS([sysexits.h])
 AC_CHECK_HEADERS([getopt.h])
-AC_CHECK_FUNC([getopt_long], [], [ AC_LIBOBJ([getopt]) AC_LIBOBJ([getopt1]) ])
 
 AC_ARG_ENABLE(
    [dmtxquery],
